packages/bootstrap-4/src/DescriptionField/DescriptionField.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { FieldProps } from "@rjsf/core";
+
+export interface DescriptionFieldProps extends Partial<FieldProps> {
+  description?: string;
+}
+
+const DescriptionField = ({ description }: Partial<FieldProps>) => {
+  if (description) {
+    return <div><h6 className="mb-5">{description}</h6></div>;
```

Comment:
We don't want descriptions to be in bold

```suggestion
    return <div><div className="mb-5">{description}</div></div>;
```